main.jsonnet

```diff
@@ -0,0 +1,172 @@
+local dex = (import '../components/dex.libsonnet') + {
```

Comment:
how is this working with `../`?

Reply:
Is this whole main.jsonnet simply unused maybe?

Reply:
Yes, this needs to be fixed and we need to add a test case that checks if this compiles correctly.